pkg/util/dbutil/table.go

```diff
@@ -15,78 +15,12 @@
 package dbutil
 
 import (
-	"context"
-	"fmt"
 	"strings"
 
-	"github.com/pingcap/errors"
-	"github.com/pingcap/tidb/pkg/ddl"
-	"github.com/pingcap/tidb/pkg/parser"
-	"github.com/pingcap/tidb/pkg/parser/ast"
 	"github.com/pingcap/tidb/pkg/parser/model"
-	_ "github.com/pingcap/tidb/pkg/planner/core" // to setup expression.EvalAstExpr. See: https://github.com/pingcap/tidb/blob/a94cff903cd1e7f3b050db782da84273ef5592f4/planner/core/optimizer.go#L202
-	"github.com/pingcap/tidb/pkg/types"
 	_ "github.com/pingcap/tidb/pkg/types/parser_driver" // for parser driver
-	"github.com/pingcap/tidb/pkg/util/collate"
 )
 
-func init() {
-	collate.SetNewCollationEnabledForTest(false)
-}
```

Comment:
see https://github.com/pingcap/tidb/pull/52191#issuecomment-2024836481

after we reuse interface in this pkg in lightning, this `init` might cause some test fail